lib/storage/base.js

```diff
@@ -105,7 +105,9 @@ export class StorageBase {
                     `Unique subhash '${result.uniqueSubHash}' ` +
                         `(${result.alreadyPresent ? 'was already present' : 'newly-created'})`,
                 );
-                if (!result.alreadyPresent) {
+                if (result.alreadyPresent) {
```

Comment:
This might conflict with #4499, happy to resolve merge conflicts in that pr to not further complicate things here

Reply:
Thanks jeremy, but this can wait too...it's nearly all mechanical stuff here...